backend/src/projektiSearch/dynamoDBStreamHandler.ts

```diff
@@ -38,18 +39,27 @@ async function handleManagementAction(event: MaintenanceEvent) {
   if (action == "deleteIndex") {
     await new ProjektiSearchMaintenanceService().deleteIndex();
   } else if (action == "index") {
-    const startKey = await new ProjektiSearchMaintenanceService().index(event);
-    if (startKey && startKey !== event.startKey) {
-      const newEvent: MaintenanceEvent = { action: "index", startKey };
+    let scanResult;
+    const events: MaintenanceEvent[] = [{ action: "index" }];
+    do {
+      scanResult = await projektiDatabase.scanProjektit(scanResult?.startKey);
+      if (scanResult.startKey) {
+        const newEvent: MaintenanceEvent = { action: "index", startKey: scanResult.startKey };
+        events.push(newEvent);
+      }
+    } while (scanResult.startKey);
```

Comment:
Typescript valittaa, että:
_Property 'startKey' does not exist on type 'never'.ts(2339)_
Ja niinhän tosiaan ekassa loopissa onkin.
```suggestion
    let startKey: string | undefined = undefined;
    const events: MaintenanceEvent[] = [{ action: "index" }];
    do {
      startKey = (await projektiDatabase.scanProjektit(startKey)).startKey;
      if (startKey) {
        const newEvent: MaintenanceEvent = { action: "index", startKey };
        events.push(newEvent);
      }
    } while (startKey);
```